README.md

```diff
@@ -44,11 +44,6 @@ services:
       - "4173:4173"
     volumes:
       - ./data:/iceberg/data
-    # healthcheck:
-    #   test: ["CMD", "curl", "-f", "http://localhost:8080/"]
```

Comment:
We can make healthcheck in api 😄

Reply:
Yeah I removed :) It can also be added to Dockerfile too. Backend can add a `/health` endpoint though at some point to do a more extensive check.

For now I dropped it as its something we can handle later